src/index.jsx

```diff
@@ -8,9 +8,14 @@ export default (languageDefinitions) => {
     Lowlight.registerLanguage(language, definition);
   });
 
-  const Code = ({ className, children }) => (
-    <Lowlight language={className.split('-')[1]} value={children} />
-  );
+  const Code = ({ className = '', children }) => {
+    const language = className.split('-')[1] || '';
```

Comment:
Not adding `sanitize` property to remark-react' options, `props.className` is empty string `''`. At this time, `''.split(-')[1]` is `undefined`. That's why I assign `''` to `language` as default value.